README.md

```diff
@@ -1,47 +1,176 @@
-  container: 'local1'
-  primary: 'local2'
-  text: 'local3'
-###
+## Installing
+`npm install --save react-e`
+
+
+## Features
+
+React-e is a drop in replacement for `react.createElement`, but it provides syntactic advantages:
+
+#### `props` can be omitted even if you have children:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre lang="javascript">
+e('div',
+  e('span', 'Hello world'))
+    </pre></td>
+    <td><pre lang="javascript">
+createElement('div', {},
+  createElement('span', {}, 'Hello world'))
+    </pre></td>
+    <td><pre lang="jsx">
+&lt;div&gt;
+  &lt;span&gt;Hello world&lt;/span&gt;
+&lt;/div&gt;
+    </pre></td>
+  </tr>
+</table>
+
+#### static classnames can be provided inline using jade syntax:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre lang="javascript">e('label.foo.bar')</pre></td>
+    <td><pre lang="javascript">createElement('label', {className: 'foo bar'})</pre></td>
+    <td><pre lang="jsx">&lt;label className="foo bar"&gt;&lt;/label&gt;</pre></td>
+  </tr>
+</table>
+
+#### `div` is assumed if tag is not provided:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre lang="javascript">e('.foo')</pre></td>
+    <td><pre lang="javascript">createElement('div', {className: 'foo'})</pre></td>
+    <td><pre lang="jsx">&lt;div className="foo"&gt;&lt;/div&gt;</pre></td>
+  </tr>
+</table>
+
+#### [classnames](https://www.npmjs.com/package/classnames) module is built-in:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre lang="javascript">
+e('.foo', {
+  className: {active: this.state.isActive}
+})
+    </pre></td>
+    <td><pre lang="javascript">
+classnames = require('classnames')
+
+createElement('div', {
+  className: classnames('foo', {active: this.state.active})
+})
+    </pre></td>
+    <td><pre lang="jsx">
+classnames = require('classnames')
+
+&lt;div
+  className={classnames('foo', {active: this.state.active})}
+&gt;&lt;/div&gt;
+    </pre></td>
+  </tr>
+</table>
+
+#### [css modules](https://github.com/css-modules/css-modules) are bound when using `require('react-e/bind')(styles)`:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre lang="javascript">
+styles = require('./stylesheet.css')
 e = require('react-e/bind')(styles)
 
-e '.container',
-  e 'button.primary', 'Click me'
-  e 'span.text', className: active: true, 'Some text'
-```
-Is equivalent to:
-```jsx
-<div className="local1">
-  <button className="local2">Click me</button>
-  <span className="local3 active">Some text</span>
-</div>
-```
+e('.foo', {className: someVar})
```

Comment:
It would be a variable containing some css classname. Would a better variable name or a `someVar = "some-class"` at the top of the file make it more clear?